packages/alchemy/src/react/hooks/useExportAccount.ts

```diff
@@ -0,0 +1,84 @@
+"use client";
+
+import { useMutation, type UseMutateFunction } from "@tanstack/react-query";
+import { createElement, useCallback } from "react";
+import { DEFAULT_IFRAME_CONTAINER_ID } from "../../config/createConfig.js";
+import type { ExportWalletParams as ExportAccountParams } from "../../index.js";
+import { useAlchemyAccountContext } from "../context.js";
+import type { BaseHookMutationArgs } from "../types.js";
+import { useSigner } from "./useSigner.js";
+
+export type UseExportAccountData = boolean;
+
+export type UseExportAccountParams = void;
+
+export type UseExportAccountMutationArgs = {
+  params?: ExportAccountParams;
+} & BaseHookMutationArgs<UseExportAccountData, UseExportAccountParams>;
+
+export type ExportAccountComponentProps = {
+  iframeCss?: string;
+  className?: string;
+  isExported: boolean;
+};
+
+export type UseExportAccountResult = {
```

Comment:
the code here in this hook assumes web environment.
Can we make it generic so that the client can also be non-web react environment but also react native? (i.e. webview)

The API between webview <> iframe should be similar/compatible, but haven't looked deeply myself yet if having a shared react hook for this hook between web vs native env would work/be better.